server/base/src/main/java/org/apache/accumulo/server/util/ManagerMetadataUtil.java

```diff
@@ -177,45 +175,11 @@ private static KeyExtent fixSplit(ServerContext context, TableId tableId, Text m
     }
   }
 
-  /**
-   * Update the last location if the location mode is "assignment". This will delete the previous
-   * last location if needed and set the new last location
-   *
-   * @param context The server context
-   * @param tabletMutator The mutator being built
-   * @param location The new location
-   * @param lastLocation The previous last location, which may be null
-   */
-  public static void updateLastForAssignmentMode(ClientContext context,
-      Ample.TabletUpdates<?> tabletMutator, TServerInstance location, Location lastLocation) {
+  public static void updateLastLocation(Ample.TabletUpdates<?> tabletMutator,
```

Comment:
This ManagerMetadataUtil code is only used by old tablet server split recovery code that needs to be migrated to upgrade code eventually.  It would be nice to move this static method to AbstractTabletStateStore as that is the only thing that uses (except for unit test which could be renamed).  

Could also inline updateLocation into this method as this method is the only thing that calls it.

Reply:
Moved method in a26fec3